src/backend/gporca/libgpopt/src/operators/CExpressionPreprocessor.cpp

```diff
@@ -208,6 +208,7 @@ CExpressionPreprocessor::PexprSimplifyQuantifiedSubqueries(CMemoryPool *mp,
 	GPOS_CHECK_STACK_SIZE;
 	GPOS_ASSERT(NULL != mp);
 	GPOS_ASSERT(NULL != pexpr);
+	GPOS_ASSERT(NULL != (*pexpr)[0]);
```

Comment:
If` CUtils::FQuantifiedSubquery(pop)` is false then we don't need to check `(*pexpr)[0]`.